### ISSUE TITLE
Qualitative Data Request

### ISSUE THREAD

**Issue by outerpress** [department-of-veterans-affairs/va.gov-team#99595]
### Data Type Requested

User Surveys

### Data Source Type

Medallia

### Date Range Start

12-15-2024

### Date Range End

12-21-2024

### Product Name(s)

Appointments FE

### Product Description

A tool for Veterans to schedule and manage their appointments online.

### Survey URL(s), key term(s), case category or type

All comments under and including these URLS and all of their sub-URLs:

- https://www.va.gov/health-care/schedule-view-va-appointments/
- https://www.va.gov/my-health/appointments

### Sub URLs

Yes

### File Delivery Format

All in one file, on one tab

### Word Cloud

None

### VA Email Address(es)

Peter.Russo1@va.gov

### Add Assignee and Labels

If not already done so, please ensure @newworld2616 is set as the assignee for this issue and "VSP-contact-center" and "cc-data-request" are set as labels.
Please leave this field as is when submitting.

**Comment by newworld2616** [comment]
Good morning, I have shared 2 files. One of the data is from Medallia site. The other one is from AWS S3 bucket. Some stakeholders were complaining about missing data. Let me know which file works best for your team. Thank you.

**Comment by outerpress** [comment]
Hello @newworld2616 Workbook 99595 seems to contain more comments, and includes all the comments from  #9959MedalliaSlte. The only exception is that the "Medallia" sheet contains comments from 12/15 where the other does not. So I prefer the one with more comments, 99595. How do I note that in my requests?

Also do both of these questions/columns in the data reflect CSAT?:
* VA.Gov Experience
* Please rate your experience with this site today?

Thank you!

Also fyi @rbsingleton @AnneCostello

**Comment by newworld2616** [comment]
Hello, 12/15 was included in last weeks upload. Starting new year, we were told, we should get all the data on the Medallia site too.
I will share the data from S3 from now. Once the fix is done, I will use the Medallia site. Appreciate your feedback. Happy Holidays!

**Comment by outerpress** [comment]
Sounds good, and happy holidays to you - thank you for all the great data this year!
